.test-infra/jenkins/job_beam_PostCommit_Java_Nexmark_Direct.groovy

```diff
@@ -0,0 +1,80 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import common_job_properties
+
+// This job runs the suite of ValidatesRunner tests against the Direct runner.
+job('beam_PostCommit_Java_Nexmark_Direct') {
+  description('Runs the Nexmark suite on the Direct runner.')
+
+  // Set common parameters.
+  common_job_properties.setTopLevelMainJobProperties(delegate, 'master', 240)
+
+  // Sets that this is a PostCommit job.
+  common_job_properties.setPostCommit(delegate)
+
+  // Allows triggering this build against pull requests.
+  common_job_properties.enablePhraseTriggeringFromPullRequest(
+    delegate,
+    'Apache Direct Runner Nexmark Tests',
+    'Run Direct Nexmark')
+
+  // Gradle goals for this job.
+  steps {
+    shell('echo *** RUN NEXMARK IN BATCH MODE USING DIRECT RUNNER ***')
+    gradle {
+      rootBuildScriptDir(common_job_properties.checkoutDir)
+      tasks(':beam-sdks-java-nexmark:run')
+      common_job_properties.setGradleSwitches(delegate)
+      switches('-Pnexmark.runner=":beam-runners-direct-java"' +
+              ' -Pnexmark.args="' +
+              '        --bigQueryTable=nexmark\n'+
+              '        --bigQueryDataset=nexmark\n'+
+              '        --project=apache-beam-testing\n'+
+              '        --resourceNameMode=QUERY_RUNNER_AND_MODE\n'+
+              '        --exportSummaryToBigQuery=true\n'+
```

Comment:
This in effect will cause that "phrase triggered" runs' results (from PR's branch) and periodic runs' results (from master) will get mixed in the same BigQuery table. If I'm not wrong (please correct me if I am), we should not keep the results from Phrase triggered jobs in the table (so for such jobs `exportSummaryToBigQuery` should be false).  This is the same problem that we have in Performance tests. Unfortunately, it was not addressed yet. See [here](https://issues.apache.org/jira/browse/BEAM-4298) for more context if you're interested.  

I'm leaning to splitting the jobs to separate ones, as Kenneth suggests. What is more, you can even place all the job definitions in one .groovy file, see [job_PerformanceTests_FileBasedIO_IT.groovy](https://github.com/apache/beam/blob/master/.test-infra/jenkins/job_PerformanceTests_FileBasedIO_IT.groovy) as an example on how to define jobs parametrized way. Not only this fixes the the problem but is also concise. Alternatively, we can skip this for now but create a JIRA for later.

WDYT @echauchot (& @kennknowles )?

Reply:
Is this expanding the scope of this PR too much? Perhaps an alternative to keep the data clean is to simply not have phrase triggering at all, making it pure postcommit?

Reply:
@lgajowy what you say about the mix is right. What we can do for now is to disable phrase triggering (once the PR works to be able to test) to have only the postCommits in BQ tables. One day if triggering from PR is needed, then we'll add the jobs and deduplicate the code.